<?xml version="1.0"?>

<launch>
    <node name="platform_detection_real" pkg="uav_vision" type="platform_detection_real" output="screen">
    
        <!--COLOUR SELECTION-->
        <!-- GREEN -->

        <param name="H_min" value="36"/>
        <param name="H_max" value="70"/>
        <param name="S_min" value="25"/>
        <param name="S_max" value="255"/>
        <param name="V_min" value="25"/>
        <param name="V_max" value="255"/>

        <!--TARGET SIZE FOR TESTING-->
        <param name="teoric_size" value="0.032004"/> <!--Value in m2, default is 0.64-->
        <param name="altura_platform" value="0.01"/> <!--Value in m, default is 0.392-->

        <!-- PUBLISHED TOPIC -->
        <param name="platform_position_topic" value="/platform_xl/estimated_position"/>

        <!-- TELLO TOPICS -->
        <!-- CAMERA TOPICS -->
        <param name="image_topic" value="/tello/camera/image_raw"/>
        <param name="cam_info_topic" value="/tello/camera/camera_info"/>
        <!-- HEIGHT AND IMU TOPIC -->
        <param name="tello_altitude" value="/tello/status/height_m"/>
        <param name="imu_topic" value="/tello/imu"/>
    </node>
 </launch>
